package.json

```diff
@@ -11,14 +11,23 @@
     "type": "git",
     "url": "git://github.com/easypost/easypost-node.git"
   },
-  "main": "index.js",
+  "main": "src/index.js",
   "types": "types/index.d.ts",
   "license": "MIT",
   "engines": {
     "node": ">= 12.0"
   },
+  "exports": {
+    ".": {
+      "require": "./dist/cjs/index.js",
+      "import": "./dist/esm/index.js"
+    }
+  },
```

Comment:
question: I assume this replaces the `index.js` file in the root? This accomplishes the same results though allowing users to require the entire library?

Reply:
Yea, exactly. Starting in Node v12, you can use the [conditional exports](https://nodejs.org/docs/latest-v22.x/api/packages.html#conditional-exports) to specify what they should get when they use your package. Basically for both the CJS and ESM builds, the entire src directory is built and built in these folders, so if they are using an `import EasyPostClient from "@easypost/api"` it will import the "import"/esm build, and for older projects that do a `const EasyPostClient = require("@easypost/api")` it will use the "require"/cjs built assets.